### PR TITLE
Release 0.17.0

### PR DESCRIPTION
**Pull request by oguzhanunlu**

### CODE REVIEW

**Comment by oguzhanunlu** [comment]
thanks @benjben , I tested the script itself inside a latest gcp mini and it was fine, we should check that the next release (rc1 I think) should come with index patterns out of the box, if not, we shouldn't proceed with the release and fix it but I believe this fix will be enough

**Comment by benjben** [comment]
I now realize that due to the testing workflow of Mini (need to release an RC and do manual deployment and tests), it's probably better to stick to the old way of putting everything in the `release/` branch and merging to `master`.

**Comment by ScullWM** [comment]
Any release date for this version ?

**Comment by benjben** [comment]
Hi @ScullWM , hopefully this week !

**Review by istreeter — COMMENTED** [review]
Looks good!

How much effort would it be to change to fs2-based enrich?  Feels wrong that we'll be deprecating stream-enrich whilst we're still using it in Mini.  On the other hand I don't want to delay this release any further.

File: .github/workflows/publish.yml [hunk 1]
```diff
@@ -38,12 +27,15 @@ jobs:
       - name: Extract tag from ref
         if: startsWith(github.ref, 'refs/tags/')
         id: get_tag
-        run: echo ::set-output name=VERSION::$(echo $GITHUB_REF | sed "s/refs\/tags\///")
+        run: echo "VERSION=$(echo $GITHUB_REF | sed "s/refs\/tags\///")" >> $GITHUB_OUTPUT
```
Comment (istreeter) [inline .github/workflows/publish.yml:30]: I think you can make this line a bit simpler:

```
echo VERSION=${GITHUB_REF#refs/tags/} >> $GITHUB_OUTPUT
```

Comment (istreeter) [reply .github/workflows/publish.yml:30]: Or even simpler, remove this step entirely, and then use the `github.ref_name` to tell us the tag name.  https://docs.github.com/en/actions/learn-github-actions/contexts

**Review by istreeter — APPROVED** [review]

**Pull request merged**